### ISSUE TITLE
Implement chat history

### ISSUE THREAD

**Issue by andy5995** [netpanzer/netpanzer#127]

**Comment by winrid** [comment]
I like the "console" menu that quake like games have, which would also show you chat log. You open it via tilda key.

**Comment by andy5995** [comment]
@0ad and @MegaGlest have chat history implemented if you're looking for any other ideas.

**Comment by winrid** [comment]
oh, you mean like this: 
https://wildfiregames.com/forum/uploads/monthly_2016_10/chat_history.jpg.193392bbb58c60841e7c4dd0932364e6.jpg

**Comment by andy5995** [comment]
Right. Though that doesn't really accurately portray how it looks. The window "pops up" in the foreground of your game screen.

**Comment by winrid** [comment]
I think no matter what we have it should be able to be controlled via the keyboard (sending messages, opening it, dismissing it).
